src/App.vue

```diff
@@ -125,10 +151,10 @@ Vue.use(BootstrapVue);
 window.firebase = firebase;
 
 /**
-* This is the main entrypoint to the app.
-*/
+ * This is the main entrypoint to the app.
+ */
 export default {
-  name: 'app',
+  name: "app",
```

Comment:
I think it would be good to keep the existing lint formatting for now as this is failing the build, might be cause your editor has a default setup. If we can get just your changes passing the build for now, then we can look into cleaning up the formatting in another PR. Easier to see your changes that way :)

Reply:
Just switched up. My bad, I had accidentally left auto format on. Should be fixed now. Used eslint-disable-next-line on a few lines though so let me know if that's an issue.